rllib/algorithms/dqn/dqn_rainbow_learner.py

```diff
@@ -0,0 +1,72 @@
+import abc
+
+from typing import Any, Dict
+
+from ray.rllib.algorithms.dqn.dqn import DQNConfig
+from ray.rllib.core.learner.learner import Learner
+from ray.rllib.utils.annotations import override
+from ray.rllib.utils.metrics import LAST_TARGET_UPDATE_TS, NUM_TARGET_UPDATES
+from ray.rllib.utils.typing import ModuleID
+
+# Now, this is double defined: In `SACRLModule` and here. I would keep it here
```

Comment:
I think it's fine to leave them here as duplicates for better readability.

Imo, it's really not good to cross-import between different algorithms. Everywhere else, this would be ok to share.